src/pages/prescreen-1b.tsx

```diff
@@ -58,207 +67,284 @@ class Prescreen1b extends React.Component<Prescreen1bProps> {
+                    <LabelText>PDF (MySocialSecurity)</LabelText>
+                  </AnswerBox>
+                  <AnswerBox>
+                    <RadioButton
+                      type="radio"
+                      name="earningsFormat"
+                      value={EarningsEnum.PDFPRINT}
+                      onChange={() => setEarningsFormat(EarningsEnum.PDFPRINT)}
+                      checked={earningsFormat === EarningsEnum.PDFPRINT}
+                    />
+                    <LabelText>PDF (scanned from print)</LabelText>
+                  </AnswerBox>
+                  <AnswerBox>
+                    <RadioButton
+                      type="radio"
+                      name="earningsFormat"
+                      value={EarningsEnum.PAPER}
+                      onChange={() => setEarningsFormat(EarningsEnum.PAPER)}
+                      checked={earningsFormat === EarningsEnum.PAPER}
+                    />
+                    <LabelText>Paper (mailed from SSA)</LabelText>
+                  </AnswerBox>
                 </Card>
-              </HowToContainer>
-              <HowToContainer>
+              ) : null}
+              {haveEarnings === false ? (
                 <Card>
-                  <H2>HOW-TO</H2>
-                  <Link href="https://secure.ssa.gov/RIL/SiView.action">Signup or login to your online account at MySocialSecurity</Link>
+                  <QuestionText>
+                    Do you have a MySocialSecurity account?
+                  </QuestionText>
+                  <AnswerBox>
+                    <RadioButton
+                      type="radio"
+                      name="haveSSAAccount"
+                      value="true"
+                      onChange={() => setHaveSSAAccount(true)}
+                      checked={haveSSAAccount === true}
+                    />
+                    <LabelText>Yes</LabelText>
+                  </AnswerBox>
+                  <AnswerBox>
+                    <RadioButton
+                      type="radio"
+                      name="haveSSAAccount"
+                      value="false"
+                      onChange={() => setHaveSSAAccount(false)}
+                      checked={haveSSAAccount === false}
+                    />
+                    <LabelText>No</LabelText>
+                  </AnswerBox>
                 </Card>
-              </HowToContainer>
-            </> : null
-          }
-          </ContentContainer>
-          <GlosaryStyle>
-          <Glossary 
-          title="MYSOCIALSECURITY"
-          link="https://www.ssa.gov/myaccount/"
-          linkText="Login or signup online for a MySocialSecurity using this link."
-          >
-          MySocialSecurity is the Social Security Administrations online service. With a MySocialSecurity account , you can download a copy of your earnings record to use for this question.
-          </Glossary>
-          {
-            this.showFileUpload() == true && <Glossary
```

Comment:
These got deleted accidentally I think

Reply:
I added them back

Reply:
If you want to update the current images, please feel free to update them. The resolution could be better.

Reply:
If you want to update the current images in a carousel, please feel free to update them. The resolution could be better.